cookbooks/aws-parallelcluster-environment/resources/efs/partial/_mount_umount.rb

```diff
@@ -40,9 +40,9 @@
 
     # See reference of mount options: https://docs.aws.amazon.com/efs/latest/ug/automount-with-efs-mount-helper.html
     mount_options = "_netdev,noresvport"
-    if efs_encryption_in_transit == "true"
+    if efs_encryption_in_transit == "true" || efs_encryption_in_transit == true
```

Comment:
I think the problem is just in update, since the dna.json doesn't have booleans, just strings and the update generates the data.  It seems adding an EFS fs with encryption in transit wasn't tested, and is an edge case since no one has reported an issue until now.

Reply:
Either converting a string to a boolean or checking for the string and the boolean seems the same to me.  Why is the former better than the latter?

Reply:
But considering we use the data from dna.json throughout the codebase, we should force these to be strings to match that pattern